src/quoteoftheday/routes.py

```diff
@@ -41,6 +39,24 @@ def index():
 
     return render_template("index.html", **context)
 
+@bp.route("/heart", methods=["POST"])
+def heart():
+    user = "Guest"
+    if current_user.is_authenticated:
+        user = current_user.username
+    
+    # Get the heart action from the request data
+    data = request.get_json()
+    action = data.get('action', 'like')  # Default to like if not specified
+    
+    # Track the appropriate event based on the action
+    if action == 'like':
+        track_event("Liked", user)
+    elif action == 'unlike':
+        track_event("Unliked", user)
```

Comment:
We don't send the unlike event.

Reply:
We should now, see line 44 of index.html